.vscode/launch.json

```diff
@@ -30,22 +29,48 @@
             "MIMode": "lldb",
         },
         {
-            "name": "Pro Debug g++",
+            "name": "Pro Debug g++ (Publisher)",
             "type": "cppdbg",
             "request": "launch",
-            "program": "${workspaceFolder}/bin/${architecture}/debug/perftest_cpp",
-            "args": ["-pub", "-domain", "0", "-latencyCount", "1", "-dataLen", "1000", "-latencyTest", "-multicast", "-executionTime", "2", "-noPrint"],
+            "program": "${workspaceFolder}/bin/x64Linux4gcc7.3.0/debug/perftest_cpp03",
```

Comment:
@salvacorts, make sure you change back this to the ${architecture} variable, and maybe create a copy for "Pro Debug g++ (Publisher Cpp03)"